kornia/augmentation/container/utils.py

```diff
@@ -337,13 +357,50 @@ def inverse(cls, input: torch.Tensor, module: nn.Module, param: Optional[ParamIt
                 to apply transformations.
             param: the corresponding parameters to the module.
         """
-        inverse = super().inverse(input, module, param)
+        _input = input.clone()
+
+        _input = super().inverse(_input, module, param)
 
         padding_size = cls._get_padding_size(module, param)
         if padding_size is not None:
-            inverse = cls.unpad(inverse, padding_size)
+            _input = cls.unpad(_input, padding_size)
+
+        return _input
 
-        return inverse
+
+class BBoxXYXYApplyInverse(BBoxApplyInverse):
+    """Apply and inverse transformations for bounding box tensors.
+
+    This is for transform boxes in the format [xmin, ymin, xmax, ymax].
+    """
+
+    apply_func = partial(transform_bbox, mode="xyxy", restore_coordinates=True)
+
+    @classmethod
+    def pad(cls, input, padding_size):
+        for i in range(len(padding_size)):
+            input[i, :, 0::2] += padding_size[i][0]  # left padding
+            input[i, :, 1::2] += padding_size[i][2]  # top padding
+        return input
+
+    @classmethod
+    def unpad(cls, input, padding_size):
+        for i in range(len(padding_size)):
```

Comment:
```suggestion
            for i, _ in enumerate(padding_size):
```
https://deepsource.io/gh/kornia/kornia/run/c16adc3c-beee-4661-8387-49948b333146/python/PTC-W0060/